modules/402-ingress-nginx/requirements/check_test.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 Flant JSC
+Copyright 2022 Flant JSC
```

Comment:
When you are moving files, changing the copyright is unnecessary. The copyright date means the creation date. If we can see in the git that a file was moved, it is good enough.

Reply:
You can resolve the comment when you read it. Changing this is also not required.